docs/en/writing-running-appium/web/chromedriver.md

```diff
@@ -118,7 +122,9 @@ in it. The contents of the file need to be parsable as a JSON object, like:
 }
 ```
 
-### Installing an network issues
+Since Appium 1.15.0 there is a possibility to automatically download the necessary chromedriver(s) into `chromedriverExecutableDir` from the official Google storage. The script will automatically search for the newest chromedriver version that supports the given browser/web view, download it (the hash sum is verified as well for the downloaded archive) and add to the `chromedriverChromeMappingFile` mapping. Everything, which is needed to be done from your side is to execute the server with `chromedriver_autodownload` feature enabled (like `appium --allow-insecure chromedriver_autodownload`).
+
```

Comment:
How about appending the link to https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/security.md for more details about the security flag?

Reply:
added the link